src/library-authoring/author-library/data/thunks.js

```diff
@@ -94,15 +104,23 @@ export const clearLibrarySuccess = annotateThunk(() => async (dispatch) => {
 });
 
 export const clearLibrary = annotateThunk(() => async (dispatch) => {
+  if (controllers) {
```

Comment:
apparently this function is sufficiently covered by tests, according to github. Probably the easiest way to test this is to extract the async callback function into a separate named function, write a unit test for the callback function, and then just mock controllers and check that control.abort is called when calling that function.